src/main.tsx

```diff
@@ -3,6 +3,7 @@ import ReactDOM from "react-dom/client";
 import App from "./App";
 import { DeskproAppProvider } from "@deskpro/app-sdk";
 import "./main.css";
+import "@fontsource/rubik";
```

Comment:
It's on the figma designs

Reply:
figma designs use it

Reply:
All the necessary fonts are connected from desktop-ui in ThemeProvider. No need to import anything custom for styling.

Reply:
All our apps should look the same. If there is a new font in figma, it means that it was implemented in the new version of desktop-ui. And it will be when we update the app-sdk, and only then it will be in the application.